core/src/test/scala/unit/kafka/tools/StorageToolTest.scala

```diff
@@ -213,7 +213,7 @@ Found problem:
     val availableDirs = Seq(TestUtils.tempDir(), TestUtils.tempDir(), TestUtils.tempDir()).map(dir => dir.toString)
     val stream = new ByteArrayOutputStream()
     assertEquals(0, runFormatCommand(stream, availableDirs))
-    val actual = stream.toString().split("\\r?\\n")
+    val actual = stream.toString().split("\\r?\\n").drop(1)
     val expect = availableDirs.map("Formatting %s".format(_))
     assertEquals(availableDirs.size, actual.size)
```

Comment:
It seems to me this (failed) check can be removed. All we wan to test is the folders get formatted only once, and that is verified in the following assert.

Reply:
I've addressed comments in the latest commit, thanks for reviewing.